tasks/steps/blocks/update_package.yml

```diff
@@ -43,15 +44,20 @@
   failed_when:
     - install_rpm.rc != 0
     - '"does not update installed package" not in install_rpm.results[0]'
-  when: package_info.package_type == 'rpm'
+  when: package_info.type == 'rpm'
 
 - name: 'Install DEB'
   any_errors_fatal: true
   apt:
     deb: '{{ delivered_package_path }}'
     update_cache: true
-  when: package_info.package_type == 'deb'
+  when: package_info.type == 'deb'
```

Comment:
Let's move 'Install RPM', 'Install DEB' and 'Reload systemd daemon' in block `install_rpm_or_deb.yml`